<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PermanantDialog</class>
 <widget class="QDialog" name="PermanantDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>599</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Permanent F/M</string>
  </property>
  <widget class="QLabel" name="textLabel3_9_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;i&gt;&lt;b&gt;&lt;font size=&quot;+4&quot;&gt;Permanent File&lt;/font&gt;&lt;/b&gt;&lt;/i&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>County:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="permAddress2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_2_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>210</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>City/St/Zip:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>550</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>List Only Entries Starting With:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="permName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="permZip">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="companyLabel">
   <property name="geometry">
    <rect>
     <x>363</x>
     <y>170</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Company:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="letterComboBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>550</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>290</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="SSAN">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="printLabelsButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Labels</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="permSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sa&amp;ve</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="permCity">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="printPermButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Print List</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>90</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>AC/Phone:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="permCounty">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="permAddress1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="permState">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CC #:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePerm">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>550</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3ButtonGroup" name="customerTypeButtonGroup">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>180</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Customer Type</string>
   </property>
   <widget class="QRadioButton" name="sellerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Seller</string>
    </property>
    <property name="shortcut">
     <string>Alt+S</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="buyerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Buyer</string>
    </property>
    <property name="shortcut">
     <string>Alt+B</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="permSSN">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>190</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="permPhone">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>190</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="permMortgage">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>380</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="permCCNo">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>380</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentDate">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>saleDate</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="permComment">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>570</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="permNumberLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Perm#:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3ListBox" name="customers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>850</width>
     <height>190</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans Mono</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="columnMode">
    <enum>Q3ListBox::FixedNumber</enum>
   </property>
   <property name="rowMode">
    <enum>Q3ListBox::Variable</enum>
   </property>
   <property name="variableWidth">
    <bool>true</bool>
   </property>
   <property name="variableHeight">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="permNumberEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="permMadCow">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>250</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Country Of Origin</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="permBCouncil">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Beef Council</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="permMailList">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>310</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Mailing List</string>
   </property>
  </widget>
  <widget class="QComboBox" name="relationsComboBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Owner</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Officer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Agent</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Employee</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>permName</tabstop>
  <tabstop>permAddress1</tabstop>
  <tabstop>permAddress2</tabstop>
  <tabstop>permCity</tabstop>
  <tabstop>permState</tabstop>
  <tabstop>permZip</tabstop>
  <tabstop>permCounty</tabstop>
  <tabstop>permPhone</tabstop>
  <tabstop>permSSN</tabstop>
  <tabstop>permMortgage</tabstop>
  <tabstop>permCCNo</tabstop>
  <tabstop>permComment</tabstop>
  <tabstop>permMadCow</tabstop>
  <tabstop>permBCouncil</tabstop>
  <tabstop>permMailList</tabstop>
  <tabstop>permSave</tabstop>
  <tabstop>deletePerm</tabstop>
  <tabstop>printPermButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/customerdao.h</include>
  <include location="global">vector</include>
 </includes>
 <resources/>
 <connections/>
</ui>
